src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames';
+import { Filter } from '../../types/Filter';
+
+type Props = {
+  currentFilter: string,
+  onFilterChange?: (filter: string) => void
+};
+
+export const TodosFilter: React.FC<Props> = ({
+  currentFilter,
+  onFilterChange = () => { },
+}) => {
+  return (
+    <ul className="filters" data-cy="todosFilter">
+      <li>
+        <a
+          href="#/"
+          className={classNames({
+            selected: currentFilter === Filter.ALL,
+          })}
+          onClick={() => {
+            onFilterChange(Filter.ALL);
+          }}
+        >
+          All
+        </a>
+      </li>
+
+      <li>
+        <a
+          href="#/active"
+          className={classNames({
+            selected: currentFilter === Filter.ACTIVE,
+          })}
+          onClick={() => {
+            onFilterChange(Filter.ACTIVE);
+          }}
+        >
+          Active
+        </a>
+      </li>
+
```

Comment:
you can create the array (`[Filter.ALL, Filter.ACITVE, Filter.COMPLETED]`) and map it here